masonite/snippets/scaffold/middleware.html

```diff
@@ -0,0 +1,17 @@
+""" {{ class }} Middleware """
+
+class {{ class }}Middleware:
+    """{{ class }} Middleware
+    """
+
+    def __init__(self, Request):
+        """ Inject Any Dependencies From The Service Container """
```

Comment:
Again follow the docstring here like in the core repo. Example here https://github.com/MasoniteFramework/masonite/blob/develop/app/http/middleware/AuthenticationMiddleware.py#L10-L15